### ISSUE TITLE
A few cool refraction function

### ISSUE THREAD

**Issue by Gustorn** [leios/simuleios#16]
```c++
diff = distance(lens.origin, p);
if (diff > 0.001){
    index = 1.0 / diff;
} else {
    index = 1000;
}
```

Try these in the `diff > 0.001` part of the `if` branch:

- `index = exp(diff) / diff;`
- `index = 1.0 / sqrt(diff);`
- `index = 1.0 / cbrt(diff);`
- `index = erf(1.0 / diff);`
- `index = exp(1 / cbrt(diff)) * (1 / diff);`
- `index = 1.0 / ((diff) / (diff + 1));`
- `index = 1.0 / ((diff * (6.2 * diff + 0.5)) / (diff * (6.2 * diff + 1.7) + 0.06));`

**Comment by leios** [comment]
Haha, some of these look like loads of fun! I'll try a few of these on stream tomorrow, for sure!

Thanks again for all your help!

**Comment by Gustorn** [comment]
Most of them are fairly similar, but they still look cool.

**Comment by leios** [comment]
Yeah, I will certainly be checking them out. I never really thought about doing raytracing art before... 

I also found the formula for the refractive index of the invisible lens, so I'll be trying that tomorrow too!

Super excited!

Thanks again!
